### DISCUSSION TITLE
CustomTube development going forward (regarding feature requests)

### DISCUSSION THREAD

**Discussion by lightbeam24** [lightbeam24/CustomTube#121] [Announcements]
I didn't want to have to make this announcement, but it can't be avoided any longer. Simply put: I've gotten a lot of requests for new features and improvements, like A LOT. Too many, in fact. Way more than I could have ever imagined. I will no longer be worrying about implementing those requests.

# Why?

When CustomTube released back in May, it was usable, but not even close to complete. The layouts were only half finished and were being filled to the brim with inaccuracies, and some of the code needed cleaning up. However, because I've been worried about implementing feature requests, the extension is still in more or less the same unfinished state it was in nearly 6 months ago, it just has more features now.

Basically, **this project needs focus.** I have my own roadmap for features and improvements I want to implement, and I am NOWHERE NEAR out of ideas. And really, many of the things I want to implement are things that others have asked for anyway. I just don't want to do them in the same order as others want me to.

# Bottom line

For the sake of this project's future, **development will be entirely based on my own personal roadmap going forward (basically, don't expect feature requests to be implemented).** If you're going to request anything at all that isn't a bug, make it discussion instead. If you ended up submitted a bug as a discussion or vice-versa, don't worry about it, I'll politely let you know.

I'm sorry if this announcement comes as a disappointment to anyone, but as much as I'd love to get to everyone's requests, it's simply impossible. I hope you all can understand.

# Will you ever be open to requests again?

Possibly, but not for a while.

# Can I still submit bug reports?

Yes, please do. Bug reports have been consistently helpful in CustomTube's development.

**Comment by Aleksandra0056** [comment]
So, feature requests are temporarily ignored due to a very large amount?
